mithril-aggregator/src/database/provider/single_signature/get_single_signature.rs

```diff
@@ -0,0 +1,161 @@
+use sqlite::Value;
+use uuid::Uuid;
+
+use mithril_common::{entities::Epoch, StdResult};
+use mithril_persistence::sqlite::{
+    EntityCursor, Provider, SourceAlias, SqLiteEntity, SqliteConnection, WhereCondition,
+};
+
+use crate::database::record::SingleSignatureRecord;
+
+/// Simple queries to retrieve [SingleSignatureRecord] from the sqlite database.
+pub struct GetSingleSignatureRecordProvider<'client> {
+    client: &'client SqliteConnection,
+}
+
+#[allow(dead_code)] // todo: Unused in production code, Should we keep it ?
```

Comment:
IMHO, I'd rather remove this unused code (maybe obsolete?).

Reply:
I leaved it as is in order to trigger a discussion, I also think that we should remove it but not necessary in this PR.

Reply:
This dead code should definitely be removed IMO :+1: